src/game/lara.c

```diff
@@ -1806,8 +1806,12 @@ void LaraHangTest(ITEM_INFO *item, COLL_INFO *coll)
         return;
     }
 
-    if (ABS(coll->left_floor - coll->right_floor) >= SLOPE_DIF
-        || coll->mid_ceiling >= 0 || coll->coll_type != COLL_FRONT || flag) {
+    bounds = GetBoundsAccurate(item);
+    int32_t hdif = coll->front_floor - bounds[FRAME_BOUND_MIN_Y];
+
+    if ((ABS(coll->left_floor - coll->right_floor) >= SLOPE_DIF)
+        || (coll->mid_ceiling >= 0) || (coll->coll_type != COLL_FRONT)
+        || (hdif < -SLOPE_DIF) || (hdif > SLOPE_DIF) || (flag)) {
```

Comment:
We don't need all those braces :)

Reply:
Lol it was for my sanity to read it clearly when testing.

I couldn't find any more issues in the St. Francis Folly room. I also tested the invisible crack ledges and they still work. I think it's good to merge.